client/my-sites/hosting/data-loss-warning/index.js

```diff
@@ -14,30 +14,25 @@ import './style.scss';
 import Gravatar from 'components/gravatar';
 import QueryHappinessEngineers from 'components/data/query-happiness-engineers';
 import { getHappinessEngineers } from 'state/happiness-engineers/selectors';
+import CardHeading from 'components/card-heading';
 
 const DataLossWarning = ( { avatars, translate } ) => {
 	return (
-		<div className="data-loss-warning">
+		<Card className="data-loss-warning">
 			{ ! avatars && <QueryHappinessEngineers /> }
-			<h2>{ translate( 'Unsure of what all this means?' ) }</h2>
-			<p>
-				<strong>
-					{ translate(
-						"Mistakes when editing your website's files and database can lead to data loss."
-					) }
-				</strong>
-				&nbsp;
-				{ translate(
-					'If you need help or have any questions, our Happiness Engineers are here when you need them!'
-				) }
-			</p>
-			<div className="data-loss-warning__contact">
-				<Button href={ '/help/contact/' }>{ translate( 'Contact us' ) }</Button>
+			<CardHeading>{ translate( 'Support' ) }</CardHeading>
+			<div className="data-loss-warning__avatars">
 				{ shuffle( avatars ).map( avatar => (
 					<Gravatar key={ avatar } user={ { avatar_URL: avatar } } size={ 42 } />
 				) ) }
 			</div>
-		</div>
+			<p>
+				{ translate(
+					'If you need help or have any questions, our Happiness Engineers are here when you need them.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'If you need help or have any questions, our Happiness Engineers are here when you need them!' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/atomic-hosting-layout)